applications/registration-react/src/pages/dataset-list-page/__fixtures/datasets.js

```diff
@@ -0,0 +1,276 @@
+export default {
```

Comment:
mangler instruks til å regenerere

Reply:
Notert dette for teknisk gjeld. Ekstrahere funksjon fra reducer som også kan brukes for å normalisere api-respons til fixtures.